src/main/java/seedu/weme/model/Model.java

```diff
@@ -188,18 +188,21 @@
 
     /**
      * Restores the model's Weme to its previous state.
+     * @return the feedback to give the user of the undone command.
      */
-    void undoWeme();
+    String undoWeme();
 
     /**
      * Restores the model's Weme to its previously undone state.
+     * @return the feedback to give the user of the redone command.
      */
-    void redoWeme();
+    String redoWeme();
 
     /**
      * Saves the current Weme state for undo/redo.
+     * @param feedback The string to inform the user what command was undone / redone
```

Comment:
feedback the string

Reply:
Seems like we don't have a standardized way of writing a param javadoc (Start with capital or not). Might want to make an issue of this to standardize the entire repo. I'll leave it as it is right now